apps/metabase/app.yaml

```diff
@@ -0,0 +1,83 @@
+{
+    "id": "/metabase",
+    "cmd": null,
+    "cpus": 2,
+    "mem": 2048,
```

Comment:
2 CPUs and 2 GB memory is an awful lot, does it really need that?

Reply:
The same issue comment from above says MB is designed to be vertically scalable rather than horizontally. I kinda just guesstimated this number because it's a Java app and there were some comments of people giving their instances 2-12GB RAM. It occurs to me that I should probably set the JVM options telling it to use all the RAM given to the container, lol...